### PR TITLE
Provide attribute titles from schema (language aware)

### PR DESCRIPTION
**Pull request by apepper**
Skipped `boolean` and `reference` attributes for now. They will be added later (once fully supported).

### CODE REVIEW

**Comment by apepper** [comment]
REVIEW ([2626bf5](https://github.com/Scrivito/scrivito-portal-app/pull/618/commits/2626bf5d96126059d15423f7b55d4bfe9ad1fa3a))

I added one more commit - opportunity actually has quote a large schema.

File: src/Data/Opportunity/OpportunityDataClass.ts [hunk 1]
```diff
@@ -6,13 +6,689 @@ import { pisaOpportunityDataClass } from './Pisa/pisaOpportunityDataClass'
...
+            { value: 'PSA_PRO_OPP_PSI_9', title: 'In ERP erledigt' },
+            { value: 'PSA_PRO_OPP_LIN', title: 'Lead mit Interesse' },
+            { value: 'PSA_PRO_OPP_KPI', title: 'kennt Produkt, hat Interesse' },
+            {
+              value: 'PSA_PRO_OPP_BPS',
+              title: 'hat Budget, Projekt startet in 6 Monaten',
+            },
+            {
+              value: 'PSA_PRO_OPP_DIS',
+              title: 'Anfrage uninteressant, kein Potential',
```
Comment (dcsaszar) [inline src/Data/Opportunity/OpportunityDataClass.ts:84]: * [ ] FYI I need a refresher for where/why we use opportunities in the portal. So far we only had Keyword and Description. The value here, for example, looks like internal data hat may be inappropriate in a portal context.

**Review by dcsaszar — APPROVED** [review]
Mostly fine. See comments for smaller stuff.

File: src/Data/Opportunity/OpportunityDataClass.ts (hunk 1 above)
Comment (apepper) [reply src/Data/Opportunity/OpportunityDataClass.ts:84]: => moved discussion to slack.

**Comment by apepper** [comment]
REVIEW ([154f080](https://github.com/Scrivito/scrivito-portal-app/pull/618/commits/154f080186e43ada36cafef0a466bb99b804befd))

File: src/Data/ServiceObject/ServiceObjectEditingConfig.ts [hunk 2]
```diff
@@ -7,8 +7,8 @@ provideEditingConfig(ServiceObject, {
     isCarrier: { title: 'Is carrier?' },
     isCustomer: { title: 'Is customer?' },
     isSupplier: { title: 'Is supplier?' },
-    parentId: { title: 'Parent service object (ID)' },
+    parentId: { title: 'Parent service object ID' },
```
Comment (dcsaszar) [inline src/Data/ServiceObject/ServiceObjectEditingConfig.ts:10]: * [ ] FYI some attribute names end in `Id`. With the new naming scheme we not longer explicitly highlight them.

Comment (apepper) [reply src/Data/ServiceObject/ServiceObjectEditingConfig.ts:10]: We will have do discuss this again, once we to have `reference` support.

**Pull request merged**